arex-web-api/src/main/java/com/arextest/web/api/service/controller/config/SystemConfigController.java

```diff
@@ -25,51 +30,63 @@
 public class SystemConfigController {
 
   @Resource
-  private SystemConfigRepository systemConfigRepository;
+  private SystemConfigurationRepository systemConfigurationRepository;
 
   @Resource
   private ConfigLoadService configLoadService;
 
   @PostMapping("/save")
   @ResponseBody
   public Response saveSystemConfig(@RequestBody SaveSystemConfigRequestType request) {
-    return ResponseUtils.successResponse(
-        systemConfigRepository.saveConfig(request.getSystemConfig()));
+    return ResponseUtils.successResponse(systemConfigurationRepository.saveConfig(request.getSystemConfig()));
   }
 
   @GetMapping("/list")
   @ResponseBody
   public Response listSystemConfig() {
-    SystemConfig latestSystemConfig = systemConfigRepository.getLatestSystemConfig();
-    if (latestSystemConfig == null) {
-      latestSystemConfig = new SystemConfig();
-    }
-    this.appendGlobalCompareConfig(configLoadService, latestSystemConfig);
-    return ResponseUtils.successResponse(latestSystemConfig);
+    List<com.arextest.config.model.dto.SystemConfiguration> systemConfigurations = systemConfigurationRepository.getAllSystemConfigList();
```

Comment:
can remove the full class name execution

Reply:
fixed